lib/runners/processor.rb

```diff
@@ -217,21 +168,22 @@ def add_warning_if_deprecated_version(minimum:, file: nil, deadline: nil)
     end
 
     def add_warning_if_deprecated_options(keys, doc:)
-      deprecated_keys = ci_section.symbolize_keys.slice(*keys).keys
-        .map { |k| "`" + build_field_reference_from_path("$.#{k}") + "`" }
+      deprecated_keys = ci_section.slice(*keys)
+        .select { |_key, value| value }
+        .map { |k, _v| "`" + build_field_reference_from_path("$.#{k}") + "`" }
```

Comment:
[suggestion] What about this refactoring?

```suggestion
      deprecated_keys = ci_section.slice(*keys).compact.keys
        .map { |k| "`" + build_field_reference_from_path("$.#{k}") + "`" }
```